habitat-lab/habitat/config/benchmark/nav/imagenav/imagenav_base.yaml

```diff
@@ -3,20 +3,19 @@
 defaults:
   - /habitat: habitat_config_base
   - /habitat/task: imagenav
-  - /habitat/simulator/sim_sensors:
-    - depth_sensor
-    - rgb_sensor
+  - /habitat/simulator/agents@habitat.simulator.agents.main_agent: rgbd_agent
   - _self_
 
 habitat:
   environment:
     max_episode_steps: 1000
   simulator:
-    agent_0:
-      sim_sensors:
-        rgb_sensor:
-          width: 256
-          height: 256
-        depth_sensor:
-          width: 256
-          height: 256
+    agents:
+      main_agent:
```

Comment:
Why are we changing agent_0 to main_agent? Seems less intuitive to me, especially for multi_agent support.